.changeset/forty-bats-knock.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/modal': patch
+'@twilio-paste/core': patch
+---
+
+[Modal] Remove comment for obsolete ESLint rule.
```

Comment:
suggestion (non-blocking): we can omit the changelog for Modal here as this change doesn't affect users (it's like changing a story almost)

Reply:
I chatted with @SiTaggart about this during focus time. We decided to keep the changesets to ensure we are pushing updates fully, even if they are super minor.